recipes/_install.rb

```diff
@@ -83,8 +83,13 @@ def nsp
 
 windows_password = CernerSplunk::DataBag.load(node['splunk']['windows_password'], secret: node['splunk']['data_bag_secret'])
 
+run_command = "#{node['splunk']['cmd']} help commands --accept-license --answer-yes --no-prompt"
+if Gem::Version.new(nsp['version']) >= Gem::Version.new('7.2.0')
+  run_command += " --seed-passwd 'changeme'"
+end
+#TODO: Use admin_password from databag for splunk-first-run.
```

Comment:
minor, need a space between `#` and `TODO`.  This is currently failing the build due to the rubocop offense.

Reply:
added space between # and TODO .Thanks